azurerm/internal/services/apimanagement/validate/api_management_test.go

```diff
@@ -0,0 +1,82 @@
+package validate
+
+import "testing"
+
+func TestApimSkuName(t *testing.T) {
+	tests := []struct {
+		name  string
+		input string
+		valid bool
+	}{
+		{
+			name:  "Consumption_0",
```

Comment:
Is there a reason we are no longer using the SDK constants?

Reply:
Because there is not SDK contants for `Consumption_0`, only constants `Consumption`. As the capacity ranges differ by sku type, I use a regular expression to validate the `sku_name`

Reply:
Would you mean to use SDK contants in regular expression?